### ISSUE TITLE
Problem booting after flashing Huananzhi F8 with Ultimate Patcher Tool

### ISSUE THREAD

**Issue by Mestar71** [miyconst/Mi899#121]
Dear Miyconst,

thanks to the enthusiasm woken up by watching Your excellent tutorials,
I managed to render my Huananzhi F8 unbootable by trying to activate Turbo Boost Unlock using the Ultimate Patcher Tool and your Mi899 app (fpt).
Namely, I dumped the BIOS using Mi899 app, modded it with the Ultimate Patcher Tool to unlock the Turbo Boost option, and flashed the BIOS again using Mi899 app.
Now, apart from the active CPU fan, nothing happens after powering on the system, and the LCD shows error code 79 (the particular code is not listed in the document you kindly provided on Your blog).
I am aware that You get a lot of annoying questions such as this, but I would still appreciate if You could answer a few of mine, regarding reviving the motherboard, as well.

The questions are as follows:
- is the C341a programmer compatible with the Huananzhi F8 motherboard, and do I need the one with a voltage switch, as You recommend?
- if so, which voltage is recommended and can I use the older C341a programmer with a 1.8V adapter instead?
- will I have to resolder the chip to fix the BIOS (or is the clip usable for the particular motherboard layout)?

Thanks in advance,
TM.

**Comment by miyconst** [comment]
Ultimate Patcher Tool is known to brick motherboards when BIOS is not compatible with the tool. So yeah, now the only option is to restore the BIOS with a programmer.

>is the C341a programmer compatible with the Huananzhi F8 motherboard, and do I need the one with a voltage switch, as You recommend?

The voltage according to the sepcification is 3.0-3.5V but so far I did not have a single dead BIOS chip on Chinese X99 motherboards using the original CH341a without voltage switch, in this configuraiton it supplies 5V.

>if so, which voltage is recommended and can I use the older C341a programmer with a 1.8V adapter instead?

The 1.8V adapter is required for AMD boards that use lower voltage.

>will I have to resolder the chip to fix the BIOS (or is the clip usable for the particular motherboard layout)?

I was able to read/write BIOS on all Huananzhi X99-TF/T8/F8 motherboards I tested without resoldering the BIOS chip but I can't guarantee that your motherboard is the same since Chinese always do unknown changes without any prior notificaiton.
